src/components/BenefitPlanBeneficiariesSearcher.js

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import {injectIntl} from "react-intl";
+import {formatMessageWithValues, Searcher} from "@openimis/fe-core";
+import {bindActionCreators} from "redux";
+import {connect} from "react-redux";
+import {fetchBeneficiaries} from "../actions";
+import {DEFAULT_PAGE_SIZE, ROWS_PER_PAGE_OPTIONS} from "../constants";
+import BenefitPlanBeneficiariesFilter from "./BenefitPlanBeneficiariesFilter";
+
+const BenefitPlanBeneficiariesSearcher = ({
+                                              intl,
```

Comment:
please fix indentation, but now it is not critical @jdolkowski

Reply:
done